config/secrets.yml

```diff
@@ -1,7 +1,7 @@
 default: &default
   secret_key_base: MGhvYnpIWnJaRHhLaDJrRkpOd1JqWFhHUHMuRFpVeGRhRE5GVHY1NDdBQkwxM0NkWGdnY1NnZmhNMWhCRzBBRG80U2YuTG5hMURrb3NkZ2VURkJOelZkZmhFMUM0TWJwdGFwcFJhM21hRHdQVTFoRUpWZXZCbmp3YnROZkZ0a2MK
-  http_auth_username: <%= ENV["SP_NAME"] %>
-  http_auth_password: <%= ENV["SP_PASS"] %>
+  http_auth_username: <%= Figaro.env.sp_name %>
```

Comment:
I don't know if http auth config is necessary anymore now that dashboard is deployed behind nginx like IdP is. I would suggest we drop all the http auth config, but that can be a subsequent PR when `app/controllers/application_controller.rb` is also updated.